core/src/main/java/com/hcl/ecomm/core/services/impl/MySqlServiceImpl.java

```diff
@@ -153,6 +161,44 @@ public void userComplaintUpdate(String first_Name, String last_Name, String emai
 		}
 		
 		LOG.debug("userComplaintUpdate method end.");
+		return responseCode;
+	}
+
+	@Override
+	public String userComplaintFeedback(String Status, String complaintId) {
+		LOG.debug("userComplaintFeedback method start. complaintId={} and Status={}",complaintId,Status);
+		String responseCode = StringUtils.EMPTY;
+		Connection connection = null;
+
+		try {
+			connection = getConnection();
+			PreparedStatement ps = null;
+			String update = "UPDATE complaints SET status = ? WHERE id = ?;";
+			ps = connection.prepareStatement(update);
+			ps.setString(1, Status);
+
+			ps.setString(2, complaintId);
+			int code=ps.executeUpdate();
+			if (code>0) {
+				responseCode = String.valueOf(code);
+			} else {
+				LOG.error("Something went wrong while userComplaintFeedback complaint form in mysql db.");
+			}
+
+		} catch (Exception e) {
+			LOG.error("Error while executing the sql query in userComplaintFeedback. Error = {}", e);
+		} finally {
+			try {
```

Comment:
@rakeshkumar-verma : why do we need additional try block here . wont the exception be handled by try block in line 173 ?

Reply:
@rishabh-behl  we r closing the connection in finally block and it might the case error occurred while closing the connect.